tasks/installation.yml

```diff
@@ -1 +1,72 @@
 ---
+
+- name: install icinga2 web packages
+  package:
+    name: '{{ icinga2_web_packages }}'
+    state: present
+  register: icinga2_web_register_packages_installed
+  until: icinga2_web_register_packages_installed is succeeded
```

Comment:
Can you add comment why `until` is required?

Reply:
```
[fujexo@hippaforalkus ansible-role-icinga2_web]$ ansible-lint -R -r ../lint-rules .
[405] Remote package tasks should have a retry
/home/fujexo/git/adfinis/ansible/ansible-role-icinga2_web/tasks/installation.yml:3
Task/Handler: install icinga2 web packages
```

Ansible lint complains about this. Should I add a comment to the task?

Reply:
the other solution is to disable this specific lint error and remove the `until`

Reply:
How can this be done?

Reply:
Adding an `.ansible-lint` configuration file like this:
```yaml
skip_list:
  - '405'
```

Reply:
Discussed with @karras. we will skip this lint number

Reply:
thx